command.go

```diff
@@ -193,6 +208,11 @@ type Command struct {
 	// TraverseChildren parses flags on all parents before executing child command.
 	TraverseChildren bool
 
+	// TraverseChildrenHooks will have each subcommand's PersistentPreRun and
+	// PersistentPostRun instead of overriding. It should be set on the root
+	// command.
+	TraverseChildrenHooks bool
```

Comment:
I think it's worth adding some checks to ensure that this is set in the root command only (because `traverseChildrenHooks` cares only about root command).